src/app/components/login-component/login-component.component.html

```diff
@@ -4,18 +4,18 @@
       <div class="row">
         <div class="col-md-6">
           <mat-form-field class="full-width">
-            <input matInput placeholder="Email" />
+            <input type= "email" placeholder="Email" />
           </mat-form-field>
         </div>
         <div class="col-md-6">
           <mat-form-field class="full-width">
-            <input matInput placeholder="Password" />
+            <input type="password" matInput placeholder="Password" />
           </mat-form-field>
         </div>
       </div>
     </mat-card-content>
     <mat-card-content>
-      <button mat-raised-button color="primary" class="button">Login</button>
+      <button type="button" mat-raised-button color="primary" class="button">Login</button>
```

Comment:
this change is irrevelant because the the tag is already mapped as a button "<button ......"